css-layout-classes/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<head>
+  <title>css layout classes</title>
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full" id="header">
+        <h1><strong>Choose your character!</strong></h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <img src="images/link.png" alt="link">
+      </div>
+      <div class="column-one-third">
+        <h2><strong>Super Smash Bros. Ultimate</strong></h2>
+        <p>Super Smash Bros Ultimate received universal acclaim from both critics and players, with some critics calling it the best installment in the series. It received praise for its large amount of content and fine-tuning of existing Smash gameplay elements, although its online mode was widely critisized.  </p>
+      </div>
+      <div class="column-one-third">
+        <img src="images/yoshi.png" alt="yoshi">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-fourth">
+        <img class="small" src="images/kirby.png" alt="kirby">
+      </div>
+      <div class="column-one-fourth">
+        <img class="small" src="images/fox.png" alt="fox">
+      </div>
+      <div class="column-one-fourth">
+        <img class="small" src="images/pikachu.png" alt="pikachu">
+      </div>
+      <div class="column-one-fourth">
+        <img class="small" src="images/ness.png" alt="ness">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half">
```

Comment:
A more appropriate name for this, would be `column-two-thirds`. This section is taking up more width than the image.